DeviceAPICharter.html

```diff
@@ -48,7 +48,7 @@
 
 <h1 id="title">[DRAFT] Device and Sensors Working Group Charter</h1>
 
-<p><em>This is a draft revised Device APIs Working Group (renamed Device and Sensors Working Group) charter for discussion. It has no standing. See also the <a href="https://www.w3.org/2016/03/device-sensors-wg-charter.html">charter under which the Device APIs Working Group</a> currently operates.</em></p>
+<p><em>This is a draft revised Device APIs Working Group (renamed Device and Sensors Working Group) charter for discussion. It has no standing. See also the <a href="https://www.w3.org/2016/03/device-sensors-wg-charter.html">charter under which the Device and Sensors Working Group</a> currently operates.</em></p>

```

Comment:
maybe flip the first: Device APIs Working Group (renamed Device and Sensors Working Group) -> Device and Sensors Working Group (formerly known as Device APIs Working Group); or drop the mention of Device APIs completely?

Reply:
Proposed fix https://github.com/w3c/dap-charter/pull/39/commits/69ec9e9214f4e035aaae884995cae4c03a7875c9